test/ctl/ctl_api.cpp

```diff
@@ -35,7 +35,7 @@ TEST_F(test, ctl_by_handle_os_provider) {
 
     int ipc_enabled = 0xBAD;
     ret = umfCtlGet("umf.provider.by_handle.params.ipc_enabled", hProvider,
-                    &ipc_enabled);
+                    &ipc_enabled, 0); // Some handlers omit size
```

Comment:
I don't get it - is size always required? is it a mandatory arg? when I should set it?

Reply:
Added description that is an optional argument